astropy/units/si.py

```diff
@@ -436,6 +455,20 @@
     prefixes=True,
     doc="curie: unit of radioactivity",
 )
+def_unit(
+    ["Gy", "gray"],
+    m**2 * s**-2,
+    namespace=_ns,
+    prefixes=True,
+    doc="gray: absorbed dose of ionizing radiation, kerma",
```

Comment:
I'd write "... ionizing radiation or kinetic energy released per unit mass (kerma)" - I had to look up the abbreviation.